src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
Isn't the purpose of the serialization format to prevent stuff being
deserialized by old versions of MoarVM?

On 19 May 2016 at 12:56, Jimmy Zhuo <notifications@github.com> wrote:

> In src/6model/serialization.c
> <https://github.com/MoarVM/MoarVM/pull/370#discussion_r63858307>:
>
> > @@ -10,7 +10,7 @@
> >
> >  /* Version of the serialization format that we are currently at and lowest
> >   * version we support. */
> > -#define CURRENT_VERSION 19
> > +#define CURRENT_VERSION 20
>
> I think it don't need bump it since you didn't do any version check, and
> looks like it is safe.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MoarVM/MoarVM/pull/370/files/ae8d3838635a8443f1fba0e95b2ade541cd6902c#r63858307>
>